index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Omnifood</title>
+</head>
+<body>
+    <div class="container">
+        <div class="item1" id="home1">
+            <h1 class="h-primary">A healthy meal <br> 
+                delivered to your door, <br>
+                every single day</h1>
+        </div>
+
+        <div class="item2" id="home2">
+            <p class="hero-description">The smart 365-days-per-year food subscription that will make you eat healthy again. Tailored to your personal tastes and nutritional needs.
+            </p>
+        </div>
+
+        <div class="item3" id="home3">
+            <button class="btn btn1">
+                Order Now
```

Comment:
1. add link tag to this button. 
2. Instead of 'Order Now' replace it to 'Start eating well'